src/PostgREST/Parsers.hs

```diff
@@ -146,9 +146,9 @@ pOpExpr pSVal pLVal = try ( string "not" *> pDelimiter *> (OpExpr True <$> pOper
       <|> pFts
       <?> "operator (eq, gt, ...)"
     pFts = do
-      mode <- option Normal $
-              try (string "phrase" *> pDelimiter *> pure Phrase)
-          <|> try (string "plain" *> pDelimiter *> pure Plain)
+      mode <- optionMaybe $
```

Comment:
After reading https://www.postgresql.org/docs/9.6/static/textsearch-controls.html#TEXTSEARCH-PARSING-QUERIES I'm wondering whether `to_tsquery` should be the default when no mode is specified, since

> to_tsquery offers access to more features than either plainto_tsquery or phraseto_tsquery, but it is less forgiving about its input

Too bad changing the default would cause a potentially breaking change for people who rely on our current fts behavior. Maybe I'm wrong but the plainto_tsquery function seems more likely to be useful for the strings people might pass via the url.